MobiusCore/Source/MobiusController.swift

```diff
@@ -94,18 +94,13 @@ public final class MobiusController<Model, Event, Effect> {
         )
         self.state = state
 
-        // Maps an event consumer to a new event consumer that invokes the original one on the loop queue,
-        // asynchronously.
+        // Maps an event consumer to a new event consumer that asynchronously invokes the original on the loop queue.
         //
         // The input will be the core `MobiusLoop`’s event dispatcher, which asserts that it isn’t invoked after the
-        // loop is disposed. This doesn’t play nicely with asynchrony, so here we assert when the transformed event
-        // consumer is invoked, but fail silently if the controller is stopped before the asynchronous block executes.
+        // loop is disposed. This doesn’t play nicely with asynchrony, so here we fail silently if the controller is
+        // stopped before the asynchronous block executes.
         func flipEventsToLoopQueue(consumer: @escaping Consumer<Event>) -> Consumer<Event> {
             return { event in
-                guard state.running else {
-                    MobiusHooks.errorHandler("\(Self.debugTag): cannot accept events when stopped", #file, #line)
-                }
```

Comment:
Due to this closure being created and owned by the loop—and the lifetime of the loop only being tied to the controller's running state—this doesn't appear to be a valid scenario, unless the loop reference was somehow being leaked.